bugz.py

```diff
@@ -124,20 +124,26 @@ def bugzilla_qe_verify(self):
 
                     rows.append(row)
 
-        # Create the DataFrame
-        df = pd.DataFrame(rows)
+        self.db.qa_needed_delete()
 
-        df['modification_date'] = pd.to_datetime(df['modification_date'], format='%Y%m%dT%H:%M:%S') # noqa
-        df['creation_date'] = pd.to_datetime(df['creation_date'], format='%Y%m%dT%H:%M:%S') # noqa
+        if not rows:
+            print("There are no bugs to verify today")
 
-        # Drop the columns 'type_id' and 'id'
-        df_cleaned = df.drop(columns=["type_id", "id"])
+        else:
+            # Create the DataFrame
+            df = pd.DataFrame(rows)
 
-        data_frame = self.db.report_bugzilla_qa_needed(df_cleaned)
-        self.db.report_bugzilla_qa_needed_insert(data_frame)
+            df['modification_date'] = pd.to_datetime(df['modification_date'], format='%Y%m%dT%H:%M:%S') # noqa
+            df['creation_date'] = pd.to_datetime(df['creation_date'], format='%Y%m%dT%H:%M:%S') # noqa
 
-        qe_needed_count = self.db.report_bugzilla_qa_needed_count(data_frame)
-        self.db.report_bugzilla_qa_needed_count_insert(qe_needed_count)
+            # Drop the columns 'type_id' and 'id'
+            df_cleaned = df.drop(columns=["type_id", "id"])
```

Comment:
just curious, are you trimming these columns from just the dataframe because ... not needed / housekeeping?

Reply:
yes, those are not needed, those are the id for the flag and its type which do not give much information